bin/pacbio_barcode_check.py

```diff
@@ -1,5 +1,10 @@
+#!/usr/bin/env python3
+
 """
-Notes: Forces sys.exit(1) to kill pipeline
+Pacbio Barcode Check
+------------------------
+Looks for Pacbio barcodes in ref and data.
+If supplied barcodes arn't in data then pipeline dies.
```

Comment:
The pipeline dies when a query barcode specified by the user is not found in the barcodes sequence database file (https://github.com/sanger-tol/cobiontcheck/blob/develop/third_party_files/pacbio_barcode_screen/pacbio_adaptors.fa). It doesn't die when no barcodes are found in the assembly. I think this could be made clearer here

Reply:
Ok, so if there are user-given barcodes and any of those are not found then die.
If no user-barcodes and no barcodes found, continue?

Reply:
At the minute, I am relying on the existence of `BARCODES FOUND`, now `barcodes exist` as a gatekeeping variable. So i need a more generic 'code' for cases of no-barcodes.

Reply:
Using barcodes as variable to be output, no output will kill pipeline